packages/single-line/src/SingleLineEditor.test.tsx

```diff
@@ -202,4 +202,34 @@ describe('SingleLineEditor', () => {
     expect(getByText('0 characters')).toBeInTheDocument();
     expect(getByText('Requires at least 1000 characters')).toBeInTheDocument();
   });
+
+  it('renders no validation message if withCharValidation is falsy', () => {
+    const [field] = createFakeFieldAPI((field) => {
+      return {
+        ...field,
+        type: 'Symbol',
```

Comment:
Heads up that Symbol by definition is max 256 characters. Text, on the other hand, would normally be unlimited in length unless the constraint is applied. Granted here it doesn't matter.